### PR TITLE
Added documentation

### PR DESCRIPTION
**Pull request by matthamil**
## Status
**READY**

[X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

### CODE REVIEW

**Comment by daynewright** [comment]
LGTM

**Comment by garrett-vangilder** [comment]
LGTM

File: Controllers/ProductTypesController.cs [hunk 1]
```diff
@@ -25,11 +25,21 @@ public class ProductTypesController : Controller
...
+         */
         public ProductTypesController(BangazonContext ctx)
         {
             context = ctx;
         }
 
+        /**
+         * Purpose: Creates a sorted product type list view
+         * Return:
+         *      Redirects user to product type list view
```
Comment (MattKraatz) [inline Controllers/ProductTypesController.cs]: Technically these return a view, not a redirect

File: Controllers/ProductTypesController.cs [hunk 2]
```diff
@@ -39,13 +49,27 @@ public ProductTypesController(BangazonContext ctx)
             return View(model);
         }
 
+        /**
+         * Purpose: Creates a sorted product type list that shows all products of that type
+         * Arguments:
+         *      id - product type id
+         * Return:
+         *      Redirects user to product type list view for a specific product type
```
Comment (MattKraatz) [inline Controllers/ProductTypesController.cs]: `Product type list view for a specific product type`

**Review by MattKraatz — APPROVED** [review]
Quick semantic change

**Comment by Anulfo** [comment]
LGTM

**Pull request merged**